### PR TITLE
494 update user notifiction banner styling

### LINKED ISSUE #494: Display do not refresh msg
SW creation process is crashing many times, because people refresh the loading page. We will include a msg saying to be pacient and do not refresh until the wallet is done. 

**Design Material** https://www.figma.com/file/ZTqF54s6CSHTwAxp0U0JjQ/Circles-myxogastria?node-id=7556%3A15680

### PR DESCRIPTION
**Pull request by mikozet**

### CODE REVIEW

**Comment by louilinn** [comment]
There is no issue linked to this PR or any context provided

**Comment by louilinn** [comment]
Closes https://github.com/CirclesUBI/circles-myxogastria/issues/494

File: .env.example [hunk 1]
```diff
@@ -34,11 +34,21 @@ SENTRY_DSN_URL=
 USER_NOTIFICATION=
```
Comment (louilinn) [inline .env.example]: No customization for the universal user notification?

File: .env.example [hunk 2]
```diff
@@ -34,11 +34,21 @@ SENTRY_DSN_URL=
...
 # Customizable overrides for specific views
 USER_NOTIFICATION_DASHBOARD=
+USER_NOTIFICATION_DASHBOARD_ICON=
+USER_NOTIFICATION_DASHBOARD_COLOR=
```
Comment (louilinn) [inline .env.example]: Can we specify the functionality of these variables?

```
# Icons can be any icon that is defined in iconSelector in icons.js
# Undefined icon defaults to iconAlert (a bell)
# Colors can be any a hex such as "#A83297" or ???? 
# Color defaults to fountain blue (#48B2B7)
``

File: src/components/AppNote.js [hunk 3]
```diff
@@ -12,24 +14,53 @@ const useStyles = makeStyles(() => ({
...
+  },
+  dashboard: {
+    color: process.env.USER_NOTIFICATION_DASHBOARD_COLOR,
+    icon: process.env.USER_NOTIFICATION_DASHBOARD_ICON,
+    text: process.env.USER_NOTIFICATION_DASHBOARD,
+  },
+  default: {
+    color: theme.custom.colors.fountainBlue,
+    icon: 'IconAlert',
+    text: process.env.USER_NOTIFICATION,
```
Comment (louilinn) [inline src/components/AppNote.js]: I would change to 
```
 default: {
    color: process.env.USER_NOTIFICATION_COLOR || theme.custom.colors.fountainBlue,
    icon: process.env.USER_NOTIFICATION_ICON || 'IconAlert',
    text: process.env.USER_NOTIFICATION,
  },
  ```
  After adding those variables

**Review by louilinn — CHANGES_REQUESTED** [review]
Looks good in general!
Missing an explanation on how to use COLOR variable in .env
And customization of the general notification USER_NOTIFICATION
Not sure if necessary.

If you have too much I can do the fixes, as long as you explain what should be possible in color. Is it only hex strings? Or can we do gradient or theme colors etc?

**Comment by mikozet** [comment]
I resolved comments and update internal documentation file

**Comment by mikozet** [comment]
@louilinn for some reason this branch doesn't build for me correctly on development.

I created another PR: https://github.com/CirclesUBI/circles-myxogastria/pull/582/commits

**Pull request closed without merging**